### ISSUE TITLE
Option to ignore includes

### ISSUE THREAD

**Issue by veenarathna** [nginxinc/crossplane#19]
This is a nice to have option at least in my usecase.   Right now I am 'text' validating the config generated by controller. Instead I plan to run in through crossplane and validate the args and the directives part. Controller has some default 'include' directives but does not generate them. So when I run it through crossplane it complains about those missing includes.

**Comment by aluttik** [comment]
@veenarathna: Sorry for the late reply, I just got back from vacation.

This sounds like a good feature! :)

Here's a spec of what I have in mind:  
I'll add a `--single-file` flag or something to `parse` to disable the include functionality, as well as an `--exclude` argument that can be used to exclude certain directives entirely from the payload (which is useful for preserving privacy in some use cases). Of course these would be accessible from the python module as well.

Does that sound like it would cover your use case?

**Comment by veenarathna** [comment]
Sounds good. Would it be a ‘list' of excludes? 

thanks
-Veena

**Comment by aluttik** [comment]
Yeah, for the `--ignore` directive anyway. Check out the PR above :)

You'll probably want to use `--single-file` for your use case though, and that's just a flag.

Anyway, from the python module you can use the new features like this:
```python
crossplane.parse(filename, single=True)
crossplane.parse(filename, ignore=['foo', 'bar', 'baz'])
```

And from the CLI you can use them like this:
```
crossplane parse <filename> --single-file
crossplane parse <filename> --ignore=foo,bar,baz
```

**Comment by aluttik** [comment]
@veenarathna: I released a new version of crossplane (0.1.3). Running `pip install -U crossplane` or whatever should update it so you can use the new options.
